tasks/web.yml

```diff
@@ -24,7 +24,7 @@
 
 - name: migrate
   django_manage:
-    command: migrate
+    command="migrate --noinput -v 0"
```

Comment:
You shouldn't mix and match styles within a task.  ':' should be ok to use here, since the command is quoted.